project_stock/models/project_project.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    @api.model
+    def _get_default_picking_type(self):
+        return (
+            self.env["stock.picking.type"]
+            .search(
+                [
+                    ("code", "=", "ps_operation"),
+                    ("warehouse_id.company_id", "=", self.env.company.id),
+                ],
+                limit=1,
+            )
+            .id
+        )
+
+    @api.model
+    def _get_default_location_src_id(self):
```

Comment:
Instead of these defaults, please convert location_id and location_dest_id into computed writable fields that react when picking type is changed.